views/pages/add-org.ejs

```diff
@@ -1,5 +1,20 @@
 <!-- Navbar -->
 <%- include('../partials/header')-%>
 <!-- Page Content Starts -->
-    <h1>Add Org page</h1>
+
```

Comment:
I believe that this page should include a form that include the items mentioned in the checklist section on Trello. Since the Organization Card already exist, this form is what creates the organization cards.